proton

```diff
@@ -239,12 +239,16 @@ else:
 lfile = None
 if "SteamGameId" in env:
     if env["WINEDEBUG"] != "-all":
-        lfile_path = os.environ["HOME"] + "/steam-" + os.environ["SteamGameId"] + ".log"
+        lfile_path = os.environ["HOME"] + "/steam-proton-" + os.environ["SteamGameId"] + ".log"
         if os.path.exists(lfile_path):
             os.remove(lfile_path)
         lfile = open(lfile_path, "w+")
 else:
-    env["WINEDEBUG"] = "-all"
```

Comment:
I think you intentionally removed turning on '-all' here... Can you comment on that?

If we remove it from here, then maybe it makes sense to get rid of `env["WINEDEBUG"] != "-all"` as well?

Reply:
The ```-all``` here effectively overrides and disables all user debugging, even when specifically enabled in ```user_settings.py```.  This change allows the same debug flags to pass through as if the script were running an AppId invocation, thus making both use cases consistent in regard to logging.

You're right, the '''env["WINEDEBUG"] != "-all"''' line may be unnecessary, but I'd have to take a closer look at it.  I'm not much of a Python guy, but I can take a peek at that cleanup later today.